core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -592,47 +579,6 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     assert(manager.resourceOffer("execB", "host2", RACK_LOCAL).get.index === 1)
   }
 
-  test("do not emit warning when serialized task is small") {
```

Comment:
This may be a stretch, but `Task` *could* be big, right?  Eg. if there was a long chain of partitions, and for some reason they had a lot of data?  That seems unlikely, but I also have no idea if it ever happens or not.  It seems easy enough to add the check back in -- is there any reason not to?